Task_One/Firest_task.rb

```diff
@@ -0,0 +1,20 @@
+OPTIMAL = 110
```

Comment:
rename it to SUBTRACTOR and move inside the class

Reply:
I corrected it